app/src/main/java/org/transdroid/daemon/Utorrent/UtorrentAdapter.java

```
@@ -310,6 +323,27 @@ public DaemonTaskResult executeTask(Log log, DaemonTask task) {
 		}
 	}
 
+	private void parseJsonRemoteRssLists(JSONArray results) {
+		remoteRssChannels = new ArrayList<>();
+		RemoteRssChannel item;
+
+		for (int i = 0; i < results.length(); i++) {
+			try {
+				item = new UTorrentRemoteRssChannel(results.getJSONArray(i));
+				remoteRssChannels.add(item);
+			} catch (JSONException e) {
+				e.printStackTrace();
```

Don't ignore errors or write a comment why it is 'safe' to ignore, e.g.
`// Ignore unparseable items so we gracefully report whatever we can`